dir.targets

```diff
@@ -59,6 +59,17 @@
     </Task>
   </UsingTask>
 
+  <UsingTask TaskName="UnzipFile" TaskFactory="CodeTaskFactory" AssemblyFile="$(MSBuildToolsPath)\Microsoft.Build.Tasks.v12.0.dll">
```

Comment:
I will log an issue. About the Factory, there is another custom task defined in this file: downloadfile. I just used the same template. The end goal is to have it working on .NET Core, but I will take care of that once we have the unzip task. Sounds good?